src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -17,6 +20,9 @@ import spray.json.{DefaultJsonProtocol, DeserializationException, JsString, JsVa
 
 import scala.language.implicitConversions
 
+case class TooFewWorkersRequestedException(numberofWorkers: Int)
+  extends LeoException(s"$numberofWorkers worker(s) requested. A Standard cluster must have 2 or more workers.", StatusCodes.NotFound)
```

Comment:
see above: just say something like "because Google is terrible, 1 is not a number you can put here". and rename the exception class too

Reply:
This also covers negative numbers